lib/actuators/include/Actuator.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "mbed.h"
+
+/*
+ * The Actuator class is an abstracted class that encompasses
+ * DC motors, limit servos, and continuous servos. What "value"
+ * represents depends on the actuator type:
+ *
+ *      Actuator Type       Value
+ *      ------------------------------
+ *      DC Motor            Speed
+ *      Limit Servo         Position
+ *      Continuous Servo    Speed
+ */
+
+class Actuator {
+ public:
+  Actuator()          = default;
+  virtual ~Actuator() = default;
+
+  virtual void setValue(float value){};
+
+  virtual Actuator& operator=(float value){};
+
+  virtual float getValue() {
+    return -1;
+  }
```

Comment:
think about which of these should be pure virtual

Reply:
Is there a point to having an operator= function for the base actuator class? Does it make more sense to just implement separate operator='s in each of the derived classes instead?

Reply:
Is there a point to having an operator= function in the base Actuator class? Would it make more sense to have separate operator='s in each of the derived classes instead?

Reply:
It makes sense to have both. U may or mat not need to define them or use default, depending on rule of 3